soccer/KickEvaluator.hpp

```diff
@@ -102,6 +102,7 @@ class KickEvaluator {
      */
     std::vector<Geometry2d::Point> hypothetical_robot_locations;
 
+    static ConfigDouble* kick_std_dev;
```

Comment:
I think there is some better way to share config values between two different objects. Maybe some singleton style config class or something

Reply:
Eh. There no real point in having a singleton. Its the same as a static variable. You can do a getter function, buts its about he same.